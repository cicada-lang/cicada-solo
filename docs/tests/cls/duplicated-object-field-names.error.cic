class Test {
  a: Nat
  c: Nat
  b: Nat
}

the(Test, {
  a: 1
  b: 2
  b: 6
  c: 3
})
