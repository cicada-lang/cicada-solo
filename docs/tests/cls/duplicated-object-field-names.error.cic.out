I found syntax error in file: duplicated-object-field-names.error.cic

  6 |
  7 |the(Test, {
  8 |  a: 1
  9 |  b: 2
 10 |  b: 6
 11 |  c: 3
Found the identifier "b" in [64, 65], while expecting:
 "=":
     sequence_entry:let_the@18 -> (name: identifier)@19 ":"@20 (t: exp):operand@21 > "=" (exp: exp) optional 
 ",":
     zero_or_more:more@18 -> (head: property):field@21 > "," (tail: zero_or_more) 
 ",":
     optional:one@21 -> > "," 
 and so on ...